services/app/src/static/style/chess-board.css

```diff
@@ -1,8 +1,12 @@
 table {
-  border: 1px solid black;
   text-align: center;
 }
 
+#chess-board .white-tile,
```

Comment:
Pipeline check throws prettier error, but the code is styled accordingly to guidelines